### DISCUSSION TITLE
Support for soundbar HT-NT3

### DISCUSSION THREAD

**Discussion by Zax** [kovalev-sergey/homebridge-sony-audio#14] [General]
Some hope for soundbar support HT-NT3?
Does it change a lot from supported models?

Thanks for the work!

**Comment by kovalev-sergey** [comment]
Hi

Unfortunately Sony doesn't say anything about it. 

This plugin uses Sony Audio Control API and all devises what support it.

All list of supported devices here

https://developer.sony.com/develop/audio-control-api/documentation/supported-devices/

if sony adds support for this api to other devices, the plugin will be able to control them.

Sergey

On 13 Dec 2021, at 17:49, Paolo Zacchi ***@***.***> wrote:

﻿
Some hope for soundbar support HT-NT3?
Does it change a lot from supported models?

Thanks for the work!

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Zax** [comment]
Ty for reply!
